scripts/qa_crowbarsetup.sh

```diff
@@ -3640,6 +3640,11 @@ function get_horizon
                         ['elements']['$horizon-server']"`
     horizonserver=`resolve_element_to_hostname "$element"`
     horizonservice=`resolve_element_to_hostname "$element" service`
+    horizonprotocol=http
+    `crowbar $horizon proposal show default | \
+        rubyjsonparse "
+                    puts j['attributes']['$horizon']['apache']['ssl']"` && \
```

Comment:
/opt/dell/chef/data_bags/crowbar/template-horizon.schema declares it as
```json
"ssl": { "type": "bool", "required": true }
```

so it should not be anything else than `true` or `false`